setup/README.md

```diff
@@ -35,43 +35,50 @@ gcloud services enable storage-component.googleapis.com
 gcloud services enable ml.googleapis.com
 ```
 
-6. Create and download a service account key with the right permissions, follow the instructions [here](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#iam-service-account-keys-create-gcloud).
+6. Create and download a service account key.
+A [service account](https://cloud.google.com/iam/docs/service-accounts) is a special Google account that belongs to your application or a virtual machine (VM), instead of to an individual end user.
 
-7. Export Environment variables for your Project:
- 
-From your terminal choose from the following two options:
-
-a) Edit `setup.sh` in this directory. Set the following variables:
-    
-   - `PROJECT_ID`
-   - `BUCKET_NAME` 
-   - `GOOGLE_APPLICATION_CREDENTIALS` 
-    
-   Use `source` command to export them:
-  
-   ```bash
-     source ./setup.sh
-   ```
+In order to create a service account key, you may follow the instructions [here](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#iam-service-account-keys-create-gcloud).
+Alternatively, you may run this in your terminal, after setting the values for the 3 environment variables:
 
-b) In your terminal copy and paste the following with the corresponding edits.
 
 ```bash
-export RUNTIME_VERSION=1.13
```

Comment:
Why to delete these? These are required.

Reply:
I changed setup.sh (and therefore README) a little. Previously, you either had to modify setup.sh and update PROJECT_ID and some other variables and run it; or you needed to do it via console.

The new setup.sh allows for a hybrid solution, where you can modify some or all of the variables in setup.sh and run it, or set the variables in console and then run setup.sh (it is a bit smarter that way). setup.sh prints out all the variables that were set, as well as error messages for variables that are needed to be set.

Another reason for the change was that PROJECT_ID and GOOGLE_APPLICATION_CREDENTIALS were likely to be set in the previous steps of this README (due to Noah's comments in BugBash). In order to unify the setup process, some of these changes were necessary.

Reply:
Got it, 

PS: there is a typo on Create.